2025-05-05 23:35:58 [INFO] tests.request.test_request_api: Тест: Успешное получение деталей запроса (GET /api/request/request-id-1) [test_request_api.py:72]
2025-05-05 23:35:58 [INFO] api.request.client: Вызов GET /api/request/request-id-1 [client.py:63]
2025-05-05 23:35:58 [INFO] core.http_client: Sending GET request to /api/request/request-id-1 with params: None [http_client.py:44]
2025-05-05 23:35:58 [INFO] core.http_client: Received response 403 from http://localhost:4040/api/request/request-id-1 [http_client.py:48]
2025-05-05 23:35:58 [INFO] tests.request.test_request_api: Тест: Успешное получение деталей запроса (GET /api/request/request-id-1) [test_request_api.py:72]
2025-05-05 23:35:58 [INFO] api.request.client: Вызов GET /api/request/request-id-1 [client.py:63]
2025-05-05 23:35:58 [INFO] core.http_client: Sending GET request to /api/request/request-id-1 with params: None [http_client.py:44]
2025-05-05 23:35:59 [INFO] core.http_client: Received response 500 from http://localhost:4040/api/request/request-id-1 [http_client.py:48]
2025-05-05 23:35:59 [WARNING] utils.allure_utils: Ответ не является валидным JSON, аттачим как текст. [allure_utils.py:62]
